docker/Dockerfile

```diff
@@ -1,11 +1,14 @@
 ARG BASEIMAGE=us.gcr.io/k8s-artifacts-prod/build-image/debian-base-amd64:buster-v1.7.0
 
 FROM golang:1.16-alpine as builder
+RUN apk -U add git
```

Comment:
I think we can just switch the builder image to `golang:1.16` instead of alpine. That way we don't have to install individual packages.